### PR TITLE
Update install script

### PR DESCRIPTION
**Pull request by ipa-nhg**

### CODE REVIEW

File: .rosinstall [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    local-name: sick_visionary_t
+    version: indigo-devel
+- git:
+    uri: 'https://github.com/ipa320/sick_flexisoft.git'
+    local-name: sick_flexisoft
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_docking.git'
+    local-name: cob_docking
+    version: indigo_dev
```
Comment (ipa-fxm) [inline .rosinstall]: please sort alphabetically

File: .rosinstall [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
```
Comment (ipa-fxm) [inline .rosinstall]: what about `cob_perception_common`

Comment (ipa-nhg) [inline .rosinstall]: is it needed? could not be used release version?

Comment (ipa-fxm) [inline .rosinstall]: If your `.rosinstall` file is supposed to contain all "bringup" repositories, then we should add it...
Not much has changed in the repo, so I would be ok if you add it but comment it out...(for the sake of completeness)

Comment (ipa-nhg) [inline .rosinstall]: the initial idea was just clone cob_robots and cob_calibration_data , the repositories that had a specific robot configuration. But  we are constantly developing software, we are not so fast releasing new features and also we need some private packages... but I dont think that all the cob_bringup dependencies are needed, let me recheck the list

File: .rosinstall [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+- git:
+    uri: 'https://github.com/ipa320/cob_common.git'
+    local-name: cob_common
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_control.git'
+    local-name: cob_control
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ipa320/cob_docking.git'
```
Comment (ipa-fmw) [inline .rosinstall]: cob_docking is private so we should use the git@github... adress

File: .rosinstall [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+- git:
+    uri: 'https://github.com/ipa320/cob_robots.git'
+    local-name: cob_robots
+    version: indigo_dev
+- git:
+    uri: 'https://github.com/ros-industrial/ros_canopen.git'
+    local-name: ros_canopen
+    version: indigo-devel
+- git:
+    uri: 'https://github.com/ipa320/sick_flexisoft.git'
```
Comment (ipa-fmw) [inline .rosinstall]: private too, please use git@github.com...

**Comment by ipa-nhg** [comment]
Done!
please @ipa-fmw @ipa-fxm review again

**Comment by ipa-fxm** [comment]
see my snippet in https://github.com/ipa320/setup_cob4/issues/68#issuecomment-288480314

**Comment by ipa-nhg** [comment]
@ipa-fxm totally agree, but this pull request  only add the wstool configuration for the robot user during the installation process. It has nothing to do with catkin_make or catkin build.

before we set catkin build as standard, has to be tested on a robot, with 2 users, where one user (nhg) source the devel or the install workspace of other user (robot), testing the possibility that both users have overlays of the same packages. Also the env.sh generated by catkin build have to be checked, to be sure that the machine tags (env.sh) are working as we expect. Probably also the default bashrc on the robot has to be updated.

**Comment by ipa-fmw** [comment]
let's merge this one and address `catkin_make` vs `catkin build` in a new PR.

**Comment by ipa-fxm** [comment]
Fine with me in splitting this in a new PR
But we would need this PR soon as latest e.g. `cob_extern/indigo_dev` won't compile properly with `catkin_make`

**Pull request merged**